.gitignore

```diff
@@ -1,2 +1,3 @@
 _build/
 Gemfile.lock
+.idea
```

Comment:
Just IMHO, but that's a user-issue (so feel free to add it locally) and it doesn't really make sense to add it to this repo's `.gitignore`.